layouts/default.vue

```diff
@@ -270,4 +392,79 @@
 		background: var(--dark-500);
 		border-color: var(--dark-400);
 	}
+
+	.n-header-subj {
+		@apply dark:!text-dark-0;
+		color: #4b5563;
+	}
+
+	.btn-mark-all-as-read {
+		@apply flex h-10 items-center gap-x-2;
+		@apply bg-white;
+		@apply text-bluegray-900;
+		@apply h-10 w-full sm:w-auto;
+		@apply border border-solid !border-[var(--p-surface-300)];
+		@apply hover:!border-[var(--p-primary-500)] hover:!bg-[var(--p-primary-500)] hover:!text-[var(--bluegray-0)];
+		@apply dark:text-[var(--bluegray-0)] dark:bg-primary dark:!border-primary;
+		@apply dark:hover:!bg-[var(--p-primary-hover-color)] dark:hover:!border-[var(--p-primary-hover-color)];
+	}
+
+	.n-accordion {
+		@apply box-border flex w-full flex-col gap-y-2;
+	}
+
+	.n-accordion-panel {
+		@apply border-none dark:border dark:border-solid dark:border-[var(--table-border)];
+		@apply bg-[var(--p-surface-50)] dark:bg-dark-800;
+		@apply !p-0 !pb-4;
+	}
+
+	.n-accordion-panel[data-pc-name="accordionpanel"] {
+		@apply !rounded-xl;
+	}
+
+	.n-accordion-panel_new {
+		@apply bg-gradient-to-r from-[rgba(244,252,247,1)] to-[rgba(229,252,246,1)];
+		@apply dark:bg-none dark:!bg-[var(--dark-700)];
+	}
+
+	.n-accordion-panel_new .n-header-subj {
+		@apply !text-[var(--bluegray-900)];
+		@apply dark:!text-[var(--bluegray-0)];
+	}
+
+	.n-accordion-panel_new-circle {
+		display: inline-flex;
+		width: calc(.5rem + 2px);
+		height: calc(.5rem + 2px);
+		border-radius: 50%;
+		margin-left: calc(.5rem + 2px);
+		background-color: var(--p-primary-500);
+	}
+
+	.n-accordion-header {
+		@apply relative !p-4 !pr-8 -mb-4 text-left;
+	}
+
+	.n-accordion-content {
+		padding: 0 16px;
+		font-weight: 400;
+		line-height: 18px;
+		color: var(--bluegray-900);
+		overflow: hidden;
+		z-index: 0;
+	}
+
+	.n-expand-chevron {
+		@apply dark:!text-dark-0;
+		position: absolute;
+		top: 16px;
+		right: 16px;
+		color: var(--bluegray-900);
+		transition: all 400ms ease-out;
+	}
```

Comment:
I see 3 valid cases to move css rules to <style></style> block:
1. rules are repeated multiple times (e.g. `.sidebar-link`)
2. complex selector (e.g. `.n-accordion-content_msg p`)
3. complex css rule that is hard to specify using tailwind

In other cases, please prefer specifying tailwind rules directly at elements.